pkg/service/targetgroupsvc/update_test.go

```diff
@@ -0,0 +1,95 @@
+package targetgroupsvc
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/benbjohnson/clock"
+	"github.com/common-fate/common-fate/pkg/storage"
+	"github.com/common-fate/common-fate/pkg/targetgroup"
+	"github.com/common-fate/common-fate/pkg/types"
+	"github.com/common-fate/ddb/ddbmock"
+	"github.com/common-fate/provider-registry-sdk-go/pkg/providerregistrysdk"
+	"github.com/common-fate/provider-registry-sdk-go/pkg/providerregistrysdk/prmocks"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/golang/mock/gomock"
+)
+
+func TestUpdateTargetGroup(t *testing.T) {
```

Comment:
Based on the linear ticket here https://linear.app/common-fate/issue/CF-624/create-api-methods-for-targetgroup-and-deployment-as-per-spec update target group is not a requirement for this PR as the behaviour is not defined in our designs